cloudinit/tests/helpers.py

```diff
@@ -362,6 +362,7 @@ def reRoot(self, root=None):
             root = self.tmp_dir()
         self.patchUtils(root)
         self.patchOS(root)
+        self.patchOpen(root)
```

Comment:
This is quite the change, affects all testcases which use FilesystemMockingTestCase users.  Is there a specific reason why this change?